testsuite/integration-arquillian/tests/other/console/src/main/resources/themes/qe/login/messages/messages_en.properties

```diff
@@ -0,0 +1,4 @@
+doAccept=Yes
```

Comment:
That's a matter of perspective as far as I can tell. 

OK - Let's say this test will verify just the one property (and not all of the others from which some are missing - e.g. $termsTitle). What other test do you propose? I'll be happy to add them.